boring-sys/build/config.rs

```diff
@@ -80,6 +77,10 @@ impl Config {
             panic!("`fips` and `rpk` features are mutually exclusive");
         }
 
+        if self.features.fips_precompiled && self.features.rpk {
+            panic!("`fips-precompiled` and `rpk` features are mutually exclusive");
```

Comment:
Why are these mutually exclusive?

Reply:
Because we don't ship our precompiled FIPS package with the RPK patch applied

Reply:
Yeah I agree - though right now we panic when the `fips` and `rpk` changes are enabled, and ideally we follow what you're saying consistently throughout the codebase. For consistency's sake, I think we can leave it as is, and in a follow-up PR I'll adopt the approach you're suggesting more generally